src/zap/zapimport.cpp

```diff
@@ -2165,6 +2165,11 @@ DWORD ZapIndirectHelperThunk::SaveWorker(ZapWriter * pZapWriter)
     *(WORD *)(p + 2) = 0xC000;
     p += 4;
 
+    // orr r12, r12, #1
```

Comment:
The related runtime code should be here: https://github.com/dotnet/coreclr/blob/master/src/vm/jitinterface.cpp#L13372